launchpad/repo.bzl

```diff
@@ -235,14 +235,14 @@ cc_library(
 
 def _python_includes_repo_impl(repo_ctx):
     python_include_path = _find_python_include_path(repo_ctx)
-    python_solib = _find_python_solib_path(repo_ctx)
```

Comment:
This is based on the https://github.com/deepmind/reverb/pull/106 I did for Reverb. It's not necessary to include the .so and that actually causes troubles for some Python environments that do not install libpython.so at the system location (aka conda).